changelog.md

```diff
@@ -12,7 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 -->
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+## [2.3.4] - 2023-02-25
```

Comment:
I would recommend to mark this as feature, not only a bugfix :) 
```suggestion
## [2.4.0] - 2023-03-04
```
I've also suggest that day, I'm 80% sure to test it until then